### PR TITLE
Switching to Semver versioning

### PR DESCRIPTION
**Pull request by norkator**
Someone should provide change ideas if there's any.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,8 +73,19 @@ according to rules pr will be merged. Otherwise I ask changes. Just give me time
...
 Versioning
 ============
-Currently I just increment build number and version number itself doesn't have any logic in it.
-Maybe in the future will use Semver or similar. Not mandatory yet because haven't have need for breaking changes.
+We use [Semver versioning](https://semver.org/)
+
+```
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+MAJOR version when you make incompatible API changes,
```
Comment (ristomatti) [inline README.md:81]: when incompatible API changes are made

File: README.md [hunk 2]
```diff
@@ -73,8 +73,19 @@ according to rules pr will be merged. Otherwise I ask changes. Just give me time
...
+MINOR version when you add functionality in a backwards compatible manner, and
```
Comment (ristomatti) [inline README.md:82]: when backwards incompatible features are added

File: README.md [hunk 3]
```diff
@@ -73,8 +73,19 @@ according to rules pr will be merged. Otherwise I ask changes. Just give me time
...
+PATCH version when you make backwards compatible bug fixes.
```
Comment (ristomatti) [inline README.md:83]: when backwards compatible changes are made

File: README.md [hunk 4]
```diff
@@ -73,8 +73,19 @@ according to rules pr will be merged. Otherwise I ask changes. Just give me time
...
+```
+
+Author **Norkator** is responsible for incrementing versioning when merging pull requests, so 
+contributors don't need to manage it. If author fails on hes duties, contributors should inform
```
Comment (ristomatti) [inline README.md:87]: Note thar as a contributor you do not need to take versioning into account as the author **Norkator** is responsible of that.

hes = his

File: README.md (hunk 1 above)
Comment (norkator) [reply README.md:81]: Hmm.. it's directly from Senver website but okay

File: README.md (hunk 4 above)
Comment (norkator) [reply README.md:87]: thar -> that, fixed.

**Comment by ristomatti** [comment]
I didn't realise the texts were from a template but somehow they didn't seem to fit to this project with the original wording.

**Review by ristomatti — APPROVED** [review]

**Comment by norkator** [comment]
> I didn't realise the texts were from a template but somehow they didn't seem to fit to this project with the original wording.

True, texts are better now.

**Comment by norkator** [comment]
Need to do changes, when actually reading changes this is now wrong!

**Pull request merged**